src/main/resources/templates/disorders.html

```diff
@@ -13,12 +13,10 @@
     <header class="text-center p-3 mb-4 border-bottom">
         <div th:replace="./fragments/nav"></div>
     </header>
-
     <div class="col-lg-8 mx-auto p-4 py-md-5">
         <div class="pb-3 mb-5 text-center">
             <h1 class="text-center">Disorders</h1>
         </div>
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure